include/mbedtls/sha256.h

```diff
@@ -212,14 +219,21 @@ extern "C" {
 #endif
 
 /**
- * \brief          Output = SHA-256( input buffer )
+ * \brief          This function calculates SHA-256 
```

Comment:
Blocker: Please align to SHA-1 documentation: `This function calculates the SHA-256 or SHA-224 checksum of a buffer.`

Reply:
done